algorithms/48/fib.cpp

```diff
@@ -1,11 +1,35 @@
 #include <iostream>
+#include <bits/stdc++.h>
 using namespace std;
 
+void mult(long long A[2][2], long long B[2][2]){
+	long long q,w,e,r;
```

Comment:
Avoid using vague variable names like this

Reply:
Remade variable names to reflect matrix indices